builders/get-started/eth-compare/transfers-api.md

```diff
@@ -50,3 +52,16 @@ The following code snippet uses [`subscribeFinalizedHeads`](https://polkadot.js.
 --8<-- 'code/vs-ethereum/balance-event.md'
 
 In addition, you can find more sample code snippets related to more specific cases around balance transfers at this [GitHub page](https://gist.github.com/crystalin/b2ce44a208af60d62b5ecd1bad513bce){target=_blank}.
+
+### Using Substrate API Sidecar {: #using-substrate-api-sidecar }
+
+Developers can also retrieve Moonbeam blocks and monitor transactions using [Substrate API Sidecar](https://github.com/paritytech/substrate-api-sidecar){target=_blank}, a REST API service for interacting with blockchains built with the Substrate framework. 
```

Comment:
Highlight that the transfers are using both Substrate and Ethereum. I prefer to be clear because the title says "Substrate API Sidecar"